_data/projects/openllmetry.yml

```diff
@@ -0,0 +1,12 @@
+name: OpenLLMetry
+desc: Open-source observability for LLMs, based on OpenTelemetry
+site: https://www.opentelemetry.com
```

Comment:
Is this the right website? 

<img width="1345" alt="Screenshot 2023-12-17 at 4 07 20 PM" src="https://github.com/up-for-grabs/up-for-grabs.net/assets/359239/cbcfe08f-d30f-4a90-874c-05986dc63eff">

Reply:
Thanks for the catch 🤦‍♂️! fixed, sorry!